badge-maker/README.md

```diff
@@ -47,28 +46,19 @@ The format is the following:
 
 ```js
 {
-  text: [ 'build', 'passed' ],  // Textual information shown, in order
+  text: [ 'build', 'passed' ],  // Textual information to be shown on the badge, in order
 
-  format: 'svg',  // Also supports json
+  format: 'svg',  // One of 'svg' or 'json'
```

Comment:
'One of' is reading a bit strange to me with just two options (though I can't really put my finger on why 🤔) I don't think there's any ambiguity around what the two possible options are, and I'm happy to approve as is, but suggesting the below in case that reads a bit better to anyone else
 
```suggestion
  format: 'svg',  // Either 'svg' or 'json'
```